target/linux/brcm63xx/profiles/default.mk

```diff
@@ -11,7 +11,7 @@ define Profile/Default
   PRIORITY:=1
 endef
 
-define Profile/Default/description
+define Profile/Default/Description
```

Comment:
Here you would have to change Default, not description ...

Reply:
I left Default untouched in general as it's a special case. This here seems to be a previously introduced type I just fixed by accident...